File: slack-poster/src/main/java/dev/number6/slackposter/model/Field.java
package dev.number6.slackposter.model;

import com.google.gson.annotations.SerializedName;

public class Field {
    private String title;
    private String value;
    @SerializedName("short")
    private Boolean shortField;

    public Field(String title, String value, Boolean shortField) {
        this.title = title;
        this.value = value;
        this.shortField = shortField;
    }

    public String getTitle() {
        return title;
    }

    public String getValue() {
        return value;
    }

    public Boolean getShortField() {
        return shortField;
    }

    @Override
    public String toString() {
        return "Field{" +
                "title='" + title + '\'' +
                ", value='" + value + '\'' +
                ", shortField=" + shortField +
                '}';
    }
}

File: slack-poster/src/main/kotlin/dev/number6/slackposter/model/Field.kt
package dev.number6.slackposter.model

import com.google.gson.annotations.SerializedName

class Field(val title: String, val value: String, @field:SerializedName("short") val shortField: Boolean) {

    override fun toString(): String {
        return "Field{" +
                "title='" + title + '\'' +
                ", value='" + value + '\'' +
                ", shortField=" + shortField +
                '}'
    }

}